### PR TITLE
Clarify usage of Transparent Statements

### PR DESCRIPTION
**Pull request by yogeshbdeshpande**

### CODE REVIEW

File: draft-ietf-scitt-architecture.md [hunk 1]
```diff
@@ -201,7 +201,7 @@ Artifact:
 
 Auditor:
 
-: an entity that checks the correctness and consistency of all Transparent Statements registered by a Transparency Service (a subset of potential Transparent Statement Consumers).
+: an entity that checks the correctness and consistency of all Transparent Statements issued by a Transparency Service (to a subset of potential Transparent Statement Consumers).
```
Comment (OR13) [inline draft-ietf-scitt-architecture.md:204]: The directionality of this, feels difficult...

I agree auditor checks transparent statements... not sure the parenthetical is helpful.

Comment (OR13) [inline draft-ietf-scitt-architecture.md:204]:
```suggestion
: an entity that checks the correctness and consistency of all Transparent Statements issued by a Transparency Service.
```

Comment (OR13) [reply draft-ietf-scitt-architecture.md:204]: "issued to" will lead to the 2 party vs 3 party model discussion... we probably don't want to invite that.

File: draft-ietf-scitt-architecture.md [hunk 2]
```diff
@@ -271,7 +271,7 @@ The identity of a Transparency Service is captured by a public key that must be
 
 Transparent Statement:
 
-: a Signed Statement that is augmented with a Receipt created via Registration at a Transparency Services (stored in the unprotected header of COSE Envelope of the Signed Statement).
+: a Signed Statement that is augmented with a Receipt created via Registration in a Transparency Service(stored in the unprotected header of COSE Envelope of the Signed Statement).
```
Comment (OR13) [inline draft-ietf-scitt-architecture.md:274]:
```suggestion
: a Signed Statement that is augmented with a Receipt created via Registration in a Transparency Service ( the receipt is stored under tag TBD_N in the unprotected header of the Signed Statement).
```

File: draft-ietf-scitt-architecture.md [hunk 3]
```diff
@@ -349,12 +349,12 @@ Auditor       -->     / Collect Receipts /      /   Replay Log    /
...
+The SCITT architecture consists of a very loose federation of Transparency Services, and a set of common formats and protocols for issuing, registering Signed Statements and auditing Transparent Statements.
 
 In order to accommodate as many Transparency Service implementations as possible, this document only specifies the format of Signed Statements (which must be used by all Issuers) and a very thin wrapper format for Receipts, which specifies the Transparency Service identity and the agility parameters for the Merkle Tree Proof.
 Most of the details of the Receipt's contents are specified in the COSE Signed Merkle Tree Proof document {{-COMETRE}}.
 
-In this section, a high level the three main roles and associated processes in SCITT: Issuers and the Signed Statement issuance process, transparency Registry and the Transparent Statement Registration process, as well as  Verifiers and the Receipt validation process.
+In this section, a high level the three main roles and associated processes in SCITT: Issuers and the Signed Statement issuance process, transparency Registry and the Signed Statement Registration process, as well as Verifiers and the Receipt validation process.
```
Comment (OR13) [inline draft-ietf-scitt-architecture.md:357]:
```suggestion
This section describes, at a high level, the three main roles and associated processes in SCITT: Issuers and the Signed Statement issuance process, Transparency Service and the Signed Statement Registration process, Auditors of the Signed Statements and Transparent Statements and the Validation process (including verification of signatures).
```

File: draft-ietf-scitt-architecture.md [hunk 4]
```diff
@@ -939,7 +940,7 @@ In particular, they cannot "frame" an Issuer or a Transparency Service for Signe
 
 #### Append-only Log
 
-If a Transparency Service is honest, then a Transparent Statement including a correct Receipt ensures that the Transparent Statement passed its Registration Policy and was recorded appropriately.
+If a Transparency Service is honest, then a Transparent Statement including a correct Receipt ensures that the associated Signed Statement passed its Registration Policy and was recorded appropriately.
```
Comment (OR13) [inline draft-ietf-scitt-architecture.md:943]:
```suggestion
A Transparent Statement including a Receipt ensures that the associated Signed Statement passed its Registration Policy and was recorded appropriately.
```

**Review by henkbirkholz — CHANGES_REQUESTED** [review]
Potential topic for this Monday OR be moved to after the IETF 117 submission moratorium.

**Comment by yogeshbdeshpande** [comment]
> Potential topic for this Monday OR be moved to after the IETF 117 submission moratorium.

@henkbirkholz This is a fundamental bug in the spec, we need to correct it ASAP!

**Comment by yogeshbdeshpande** [comment]
@OR13, Thank you for the review. I agree with most of your edits.

File: draft-ietf-scitt-architecture.md (hunk 1 above)
Comment (yogeshbdeshpande) [reply draft-ietf-scitt-architecture.md:204]: ok! Agree with the statement, will change

File: draft-ietf-scitt-architecture.md (hunk 2 above)
Comment (yogeshbdeshpande) [reply draft-ietf-scitt-architecture.md:274]: I would prefer avoiding specific here. I will make it a general statement

File: draft-ietf-scitt-architecture.md (hunk 3 above)
Comment (yogeshbdeshpande) [reply draft-ietf-scitt-architecture.md:357]: Verifiers first verify the Transparent Statement, to check the inclusion in the log (offline).

Whether further Verification of Signed Statement is left to implementation, so I would remove the Signed Statement from Verifiers actions

File: draft-ietf-scitt-architecture.md [hunk 5]
```diff
@@ -440,7 +440,8 @@ Each policy that is expected to be enforced by the Transparency Service is repre
 The key of the map entry corresponds to the name of the policy, while its value (including its type) is policy-specific.
 For instance, the `register_by` policy defines the maximum timestamp by which a Signed Statement can be registered, hence the associated value contains an unsigned integer.
 
-While this design ensures that all Verifiers get the same guarantee regardless of where a Transparent Statement is registered, its main downside is that it requires the Issuer to include the necessary policies in the Envelope when the Signed Statement is produced.
+While this design ensures that all Verifiers get the same guarantee regardless from which Service
+a Transparent Statement originates from, its main downside is that it requires the Issuer to include the necessary policies in the Envelope when the Signed Statement is produced.
```
[the remark this answers is not in the document]
Comment (yogeshbdeshpande) [reply draft-ietf-scitt-architecture.md:444]: Makes sense, I will make the edits

File: draft-ietf-scitt-architecture.md (hunk 3 above)
Comment (OR13) [reply draft-ietf-scitt-architecture.md:357]: updated

File: draft-ietf-scitt-architecture.md (hunk 1 above)
Comment (henkbirkholz) [reply draft-ietf-scitt-architecture.md:204]: better?

File: draft-ietf-scitt-architecture.md (hunk 3 above)
Comment (henkbirkholz) [reply draft-ietf-scitt-architecture.md:357]: :+1:

File: draft-ietf-scitt-architecture.md (hunk 4 above)
Comment (yogeshbdeshpande) [reply draft-ietf-scitt-architecture.md:943]: This section talks about Append Only log, so the `text` about honesty makes sense here?

**Comment by yogeshbdeshpande** [comment]
@OR13 Thanks, I have done all the required edits, please have a look!

File: draft-ietf-scitt-architecture.md (hunk 1 above)
Comment (yogeshbdeshpande) [reply draft-ietf-scitt-architecture.md]: Done the change in the new commit

File: draft-ietf-scitt-architecture.md (hunk 3 above)
Comment (yogeshbdeshpande) [reply draft-ietf-scitt-architecture.md]: Done, please have a look!

File: draft-ietf-scitt-architecture.md (hunk 4 above)
Comment (OR13) [reply draft-ietf-scitt-architecture.md:950]: thats fine, I don't think the text is wrong, I just don't think it's needed here.

**Review by OR13 — APPROVED** [review]

File: draft-ietf-scitt-architecture.md [hunk 6]
```diff
@@ -272,7 +272,7 @@ The identity of a Transparency Service is captured by a public key that must be
 
 Transparent Statement:
 
-: a Signed Statement that is augmented with a Receipt created via Registration at a Transparency Services (stored in the unprotected header of COSE Envelope of the Signed Statement).
+: a Signed Statement that is augmented with a Receipt created via Registration in a Transparency Service( the receipt is stored in the unprotected header of COSE Envelope of the Signed Statement).
```
Comment (SteveLasker) [inline draft-ietf-scitt-architecture.md:275]: A space got lost:

>  a Signed Statement that is augmented with a Receipt created via Registration in a Transparency **Service (the** receipt is

File: draft-ietf-scitt-architecture.md [hunk 7]
```diff
@@ -958,8 +959,9 @@ An Auditor granted (partial) access to a Registry and to a collection of dispute
 This ensures any Verifier that trusts at least one such Auditor that (2,3) will be blamed to the Transparency Service.
 
 Due to the operational challenge of maintaining a globally consistent append-only Log,
-some Transparency Services may provide limited support for historical queries on the Transparent Statements they have registered,
-and accept the risk of being blamed for inconsistent Registration or Issuer equivocation.
+some Transparency Services may provide limited support for historical queries on the Signed Statements
```
Comment (SteveLasker) [inline draft-ietf-scitt-architecture.md:962]: nit: looks like the line breaks got reformatted. Please break on sentences.

**Review by SteveLasker — COMMENTED** [review]
A few minor nits to cleanup, then LGTM

File: draft-ietf-scitt-architecture.md (hunk 6 above)
Comment (henkbirkholz) [inline draft-ietf-scitt-architecture.md:275]:
```suggestion
: a Signed Statement that is augmented with a Receipt created via Registration in a Transparency Service (the receipt is stored in the unprotected header of COSE Envelope of the Signed Statement).
```

File: draft-ietf-scitt-architecture.md [hunk 8]
```diff
@@ -958,8 +959,9 @@ An Auditor granted (partial) access to a Registry and to a collection of dispute
...
+they have registered, and accept the risk of being blamed for inconsistent
+Registration or Issuer equivocation.
```
Comment (henkbirkholz) [inline draft-ietf-scitt-architecture.md:964]:
```suggestion
some Transparency Services may provide limited support for historical queries on the Signed Statements they have registered, and accept the risk of being blamed for inconsistent Registration or Issuer equivocation.
```

File: draft-ietf-scitt-architecture.md (hunk 7 above)
Comment (henkbirkholz) [reply draft-ietf-scitt-architecture.md:962]: approvable fix below

File: draft-ietf-scitt-architecture.md (hunk 8 above)
Comment (yogeshbdeshpande) [reply draft-ietf-scitt-architecture.md:964]: @henkbirkholz I am doing it now

**Review by henkbirkholz — APPROVED** [review]

**Review by SteveLasker — APPROVED** [review]
LGTM

Line item at 962 could be done later

**Pull request merged**